build.gradle

```diff
@@ -118,7 +118,7 @@ configure(subprojects.findAll { it.path.startsWith(':sdk:') && it.path.count(':'
                     artifact javadocJar
 
                     pom {
-                        name = project.publishName
+                        name = "Microsoft Azure Android Client Library common code For Communication Service"
```

Comment:
To release via pipeline, we also need ci.yml file. E.g. https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/communication/ci.yml

Reply:
@mitchdenny Should I be modifying the ci.yml? But at the top of the file it says "# DO NOT EDIT THIS FILE". :-) We do want to get Chat sdk published. cc: @anuchandy

Reply:
Yes you should and let's remove the `# DO NOT EDIT THIS FILE` It is not true anymore

Reply:
Changes made to ci.yml. Please take a look when you have time, @mitchdenny cc: @RezaJooyandeh @anuchandy

Reply:
ci.yml looks good to me